CLIENT/js/signupLogin.js

```diff
@@ -0,0 +1,102 @@
+const signupForm = document.getElementById('signup');
+const loginForm = document.getElementById('login');
+const signupNote = document.getElementById('signup-note');
+const btnSignUp = document.getElementById('btn-signup');
+
+const baseURL = 'http://food-fast.herokuapp.com/api/v1';
+
+const goToLogin = (event) => {
+  event.preventDefault();
+  signupForm.style.display = 'none';
+  loginForm.style.display = 'block';
+};
+
+const goToSignup = (event) => {
+  event.preventDefault();
+  loginForm.style.display = 'none';
+  signupForm.style.display = 'block';
+};
+
+const displayNote = (element, value, time, color = 'error') => {
+  setTimeout(() => {
+    element.innerText = value;
+    element.style.height = '2rem';
+    element.style.padding = '0.2rem';
+    element.className = color;
+  }, time);
+
+  setTimeout(() => {
+    element.style.height = '0rem';
+    element.style.padding = '0rem';
+  }, time + 2000);
+};
+
+const isValidEmail = value => /^\w+@\w+\.\w+$/.test(value);
+
+const isValidPassword = value => /^[A-Za-z0-9]+$/.test(value);
+
+const animateSignUpBtn = (time) => {
+  btnSignUp.innerText = '';
+  btnSignUp.className = 'btn-spinner';
+
+  setTimeout(() => {
```

Comment:
why setTimeout?

Reply:
The `btn-spinner` class creates a spin on the button to show the user that the app is busy. I need to remove it when the app is done to signal to the user that he or she can now interact with the app.